src/services/plugin.service.ts

```diff
@@ -21,6 +21,11 @@ export class PluginService {
 
   public async initPluginState(container: IContainer): Promise<void> {
 
+    if (!process.env.MONGO_DB_NAME || !process.env.MONGO_URL ||
+      !process.env.MONGO_USER_NAME || !process.env.MONGO_USER_PASS) {
+        return;
+    }
+
     if (!mongoose.connection.readyState) {
       await container.storageService.connectToDB();
     }
```

Comment:
cant we just try/catch instead

Reply:
This is for connecting to the database if not already connected. If we get to this point, we should be able to connect, and no errors should be thrown.